pkg/runtime/kubeadm_config.go

```diff
@@ -210,9 +212,35 @@ func TypeConversion(raw []byte, kind string) (interface{}, error) {
 		return nil, err
 	}
 	scheme.Default(obj)
+	OverrideDefaults(obj)
 	return obj, nil
 }
 
+func overrideKubeletDefaults(i interface{}) {
```

Comment:
```
var scheme = k8sruntime.NewScheme()
var (
	localSchemeBuilder = &kubeletconfigv1beta1.SchemeBuilder
)

func init() {
	utilruntime.Must(kubeadmv1beta2.AddToScheme(scheme))
	utilruntime.Must(kubeadmv1beta3.AddToScheme(scheme))
	utilruntime.Must(localSchemeBuilder.AddToScheme(scheme))
	utilruntime.Must(proxyconfig.AddToScheme(scheme))
	localSchemeBuilder.Register(kubeletconfig.RegisterDefaults, override)
}

func override(scheme *k8sruntime.Scheme) error {
	scheme.AddTypeDefaultingFunc(&kubelet.KubeletConfiguration{}, func(obj interface{}) {})
	return nil
}

```

this is better ?

Reply:
At my option, `fn func(interface{})` have two cons:
1. not return any error message
2. interface can be changed to any type and not under our control(like go 1.19 add generic types.)
and reflect with ptr should use with carful caution.

```golang
_ = kubeadmv1beta2.RegisterDefaults(scheme) 
_ = kubeadmv1beta3.RegisterDefaults(scheme)
_ = proxyconfig.RegisterDefaults(scheme)
```
with error handle will do better.

Reply:
Utilruntime.Must is used for error handling. I've make a little change in the recent commit, IMO, we don't need to make so much changes.